frontend/package.json

```diff
@@ -51,5 +53,10 @@
   ],
   "volta": {
     "extends": "../package.json"
+  },
+  "jest": {
+    "moduleNameMapper": {
+      "src/(.*)": "<rootDir>/src/$1"
```

Comment:
Needed so that relative path imports in our application code work when run within jest tests